### ISSUE TITLE
Unlinking Unclear

### ISSUE THREAD

**Issue by cliffweitzman** [MakeSchool-Tutorials/Tip-Calculator-Swift#32]
This portion, explaining how to link two objects to one method, from the Unlinking portion is unclear
![image](https://cloud.githubusercontent.com/assets/3460171/15979475/08120574-2f19-11e6-9f69-c2a7e05920ea.png)

I would rephrase step 4. to 
4. Since we need to recalculate the tip when billAmountField is filled and if tipSelector is changed, we will need to call the billAmountChanged function when either one of these user interface objects are changed. Thus, we want to call our billAmountField method every time either of these properties are changed. To do this we need to link both objects to our coded billAmountField function. To do this control-click (or right-click) and drag from the Tip % Selector to the billAmountField function written in your View Controller. 

I took this photo to help simplify the explanation

![screenshot 2016-06-10 14 36 28](https://cloud.githubusercontent.com/assets/3460171/15979825/580f8c34-2f1b-11e6-9f6b-a9e820273ae9.png)

**Comment by dionlarson** [comment]
Reshot all videos with new constraints (using stackview). Rewrite coming soon.
